app/services/fastboot.js

```diff
@@ -40,6 +56,11 @@ export default Ember.Service.extend({
     return this._fastbootInfo.request.host();
   }),
 
+  //response: computed(function() {
+    //return ResponseObject.create({ _response: get(this, '_fastbootInfo.response')});
+  //}),
+  response: alias('_fastbootInfo.response'),
+
```

Comment:
It works if you do a direct alias, but if you try shim it with an Ember Object, the deep linking for `statusCode` doesn't work because `statusCode` gets copied by value.